packages/snaps-controllers/src/snaps/SnapController.test.ts

```diff
@@ -5788,7 +5887,6 @@ describe('SnapController', () => {
         }),
       );
 
-      // By installing we also indirectly test that the unpacking of the file works.
```

Comment:
These should not be removed, these are actually true. You removed the wrong ones 😅

Reply:
I'm not sure how I managed to do this 😅